runtime/drivers/snowflake/sql_store.go

```diff
@@ -159,6 +160,11 @@ func (f *fileIterator) Next() ([]string, error) {
 
 	// common schema
 	schema := (*firstBatch)[0].Schema()
+	for _, f := range schema.Fields() {
+		if f.Type.ID() == arrow.TIME32 || f.Type.ID() == arrow.TIME64 {
+			return nil, fmt.Errorf("TIME type isn't supported, consider excluding it")
```

Comment:
Consider adding the column name as well, e.g. `"TIME data type (column %q) is not currently supported, consider excluding it or casting it to another data type"`